OoyalaSkinSampleApp/OoyalaSkinSampleApp/Lists/SSAIListViewController.temp_caseinsensitive_rename.m

```diff
@@ -0,0 +1,27 @@
+//
+//  SsaiListViewController.m
+//  OoyalaSkinSampleApp
+//
+//  Copyright © 2018 Ooyala, Inc. All rights reserved.
+//
+
+#import "SsaiListViewController.h"
+#import "DefaultSkinPlayerViewController.h"
+
+@interface SsaiListViewController ()
+
+@end
+
+@implementation SsaiListViewController
+
+- (void)addTestCases {
+  [self insertNewObject: [[PlayerSelectionOption alloc] initWithTitle:@"Small Player View (BigBuck+Ads)"
```

Comment:
Why not just adding this case to the `SsaiListViewController.m` instead of creating this temp file?

Reply:
I think git created the temp file when I rename the file.
The name was SssaiListViewController.m, I renamed to SSAIListViewController.m

Reply:
The file does not exist anymore. I think git create the temp file when the file was renamed.